disco/bot/bot.py

```diff
@@ -185,9 +185,20 @@ def __init__(self, client, config=None):
         for plugin_mod in self.config.plugins:
             self.add_plugin_module(plugin_mod)
 
+        # Temporary dict to prevent `dictionary changed size during iteration`
```

Comment:
Instead of doing this just call `list` on the `six.iteritems` below to materialize it.